test/test_feature2d.jl

```diff
@@ -17,7 +17,5 @@ for kp in kps
         end
     end
 
-    @test closest_match < 10
+    @test_skip closest_match < 10 # TODO This test fails intermittently
```

Comment:
Coul we fix this too in this round?

Reply:
No - it's not immediately apparent how to fix this test - except for replacing it completely with a test for SimpleBlobDetector from OpenCV, e.g. https://github.com/opencv/opencv/blob/4.x/modules/features2d/misc/java/test/SIMPLEBLOBFeatureDetectorTest.java

Also note [comment in #13](https://github.com/JuliaImages/OpenCV.jl/issues/13#issuecomment-1661637648). 

For this PR, I would like to just focus on getting the tests enabled.

Reply:
Okay, I could see why it's a bit problematic in current state. Let's merge and keep a note of the java test in the #13 open